### ISSUE TITLE
Zen Script classes?

### ISSUE THREAD

**Issue by PhilipColman** [stanhebben/MineTweaker3#332]
Some time ago I read upcoming changes to minetweaker a lot of the stuff I can not wait for one of the things be the ability to reference other zen script file, any way I found one webpage talking about this functionality no real examples, the file started with the equivalent to java packages, and i think where .zsi files, but about from that I am none the wiser if this possible now I would very much like to some fuller examples, so that I can use it mainly to share variables and method between files.

**Comment by jaredlll08** [comment]
Already been asked before. LOOK THROUGH THE ISSUE TRACKER BEFORE POSTING!

**Comment by PhilipColman** [comment]
Yes it has been closed so not visible by default and leads to an unanswered issue

**Comment by stanhebben** [comment]
The scripting improvements have been postponed (for a long time already) until I have more time to continue work on it. This includes the file includes and the classes.
